packages/front-end/components/Experiment/StopExperimentForm.tsx

```diff
@@ -100,7 +102,29 @@ const StopExperimentForm: FC<{
           onChange={(v) => {
             const result = v as ExperimentResultsType;
             form.setValue("results", result);
+
+            if (result === "dnf" || result === "inconclusive") {
+              form.setValue("excludeFromPayload", true);
+              form.setValue("releasedVariationId", "");
+              form.setValue("winner", 0);
+            } else if (result === "won") {
+              form.setValue("excludeFromPayload", false);
+              form.setValue("winner", 1);
+              form.setValue(
+                "releasedVariationId",
+                experiment.variations[1]?.id || ""
```

Comment:
We might later explore recommending a winning variation instead of assuming `variations[1]`.